technical-docs/2021-05-remote-restore-redesign.md

```diff
@@ -0,0 +1,122 @@
+# Medusa Operator - Restore redesign document
+
+## Background
+
+Medusa operator doesn’t yet support remote restores: restoring a backup to another cluster.
+As we implement this feature, we would like to have it support multiple use cases that operators could expect to have working:
+
+- Restoring a medusa-operator based backup to another Cassandra install in the same k8s cluster
+![](img/medusa_k8s-new-restore-design-1.jpg)
```

Comment:
It seems that the images do not correspond to the descriptions.

Reply:
Also the images read "K8ssandra" where I think we mean "Kubernetes cluster".

Reply:
Good catch 😅 
I'll fix this.